netlogo-headless/src/test/hubnet/server/HubNetManagerTests.scala

```diff
@@ -0,0 +1,94 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.hubnet.server
+
+import org.nlogo.api.ConfigurableModelLoader
+import org.nlogo.fileformat.{ FileFormat, NLogoFormat, NLogoHubNetFormat }
+import org.nlogo.hubnet.connection.HubNetException
+import org.nlogo.workspace.DummyAbstractWorkspace
+import org.scalatest.funsuite.AnyFunSuite
+
+class HubNetManagerTests extends AnyFunSuite {
+
+  hubnetTest("test Update View False") { (manager, connectionManager) =>
```

Comment:
Are these tests copied from somewhere?  They seem to follow different style rules than we normally use.